src/dynamic_reconfigure/parameter_generator.py

```diff
@@ -41,12 +41,10 @@
 # Need to put sane error on exceptions
 from __future__ import print_function
 
-import roslib; roslib.load_manifest("dynamic_reconfigure")
```

Comment:
What is this line needed for?

Reply:
I found the article here.
https://answers.ros.org/question/115346/what-does-roslibload_manifest-do/
http://wiki.ros.org/PyStyleGuide

I don't know this package have a dependency that is still on rosbuild or not.

Reply:
Removing it seems reasonable, but not related to this PR. I think it should be done in a separate PR.